src/ApiClient/TransferClient.ts

```diff
@@ -38,7 +37,7 @@ export default new (class TransferClient extends ApiClient {
             case 200: {
                 return new InternalStatus({
                     code: StatusCode.OK,
-                    payload: response.data as unknown as string
+                    payload: (response.data as File).name
```

Comment:
This is a base64 encoded string containg the raw data for the file, where did you get your File type from?
```suggestion
                    payload: response.data as unknown as string
```